packages/forms/src/components/Form/DatePickerInput.tsx

```diff
@@ -1,14 +1,20 @@
 import React from 'react';
 import BaseDatePickerInput, { Props } from '@airbnb/lunar/lib/components/DatePickerInput';
 import connectToForm, { ConnectToFormProps } from '../../composers/connectToForm';
-import { toString } from '../../helpers';
 
 /** `DatePickerInput` automatically connected to the parent `Form`.  */
-export function FormDatePickerInput(props: Props & ConnectToFormProps<string>) {
+export function FormDatePickerInput({
+  // Blur and focus events provided by `final-form` break the picker
+  // overlay, so let's omit them for now, as it isn't an easy fix.
+  // Luckily, we can get away with this as it's mostly used for
+  // validation purposes, and dates are deterministic.
```

Comment:
okay! thought it was something with blur/focus from the extra layer of connect to form. fair compromise!